modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -146,12 +146,14 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
                 }
             } else {
                 // allow only to view own site data
-                $site =& Site::singleton($user->getData('CenterID'));
-                if ($site->isStudySite()) {
-                    $list_of_sites = array();
-                    $user_site     =$user->getData('Site');
-                    $list_of_sites[$user->getData('CenterID')] = $user_site;
+                $site_arr = $user->getData('CenterIDs');
+                foreach ($site_arr as $key => $val) {
+                    $site[$key] = &Site::singleton($val);
+                    if ($site[$key]->isStudySite()) {
+                        $list_of_sites[$val] = $site[$key]->getCenterName();
+                    }
```

Comment:
should this be moved to the `User` class? So you could do something like `$user->getStudySites()` to make this cleaner?